### ISSUE TITLE
Remove old test site

### ISSUE THREAD

**Issue by dnguneratne** [OTRChat/NodeChat#43]
The old test site resides on the root folder. It needs to be removed cleaned up. I can work on this.

**Comment by joshghent** [comment]
@dnguneratne Yeah the heroku box currently hosts the old site. If you move the react site to the root and then change the proc file to make the new start command then it **should** work

**Comment by dnguneratne** [comment]
It seems like the Profile starts the server with `node index.js`, and express serves static pages in public.

The current version of the app has the client running separately from the express server, and accessing the server at `localhost:9876`. Wouldn't the Heroku config need to be update to facilitate this?

I was also hoping to keep the server and client separated for clarity.

![image](https://user-images.githubusercontent.com/8231763/48351825-3aeeb380-e659-11e8-9637-bbcb92431c71.png)

**Comment by joshghent** [comment]
@dnguneratne Yes you are right. The server code was moved to https://github.com/OTRChat/server
I have configured that to auto deploy the develop and master branch. The same as this repo. If you put a procfile in the root it will run. We should definitely keep all server and client code seperate.
We will need a new .env to configure the domain for the server but other than that, should be plug and play

**Comment by dnguneratne** [comment]
@joshghent, @coreyjjames I was not aware that the server is in it's own repo. Should I then remove the server folder from this repo and move everything inside client to the root folder?
